docs/release-notes/snapcraft-8-8.rst

```diff
@@ -0,0 +1,101 @@
+.. _release-8.8:
+
+Snapcraft 8.8 release notes
+===========================
+
+14 April 2025
+
+Learn about the new features, changes, and fixes introduced in Snapcraft 8.8.
+
+
+Requirements and compatibility
+------------------------------
+
+See :ref:`System requirements<system-requirements>` for information on the minimum
+hardware and installed software.
+
+
+What's new
+----------
+
+Snapcraft 8.8 brings the following features, integrations, and improvements.
+
+Cross-compiling for core24
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Previously, cross-compiling for ``core24`` snaps only worked when building on
+AMD64 for I386 or on ARM64 for ARMHF.
+
+This restriction has been dropped, so ``package-repositories`` and ``build-packages``
+keys can now define entries for any :ref:`supported architecture
+<supported-architectures>`.
+
+For more information, see the :ref:`example-cross-compiled-app` how-to.
+
+Remote build improvements
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :ref:`remote-build command <ref_commands_remote-build>` can now request builds on
```

Comment:
I can't really keep up with launchpad restrictions; I thought trying to do cross-compilation on launchpad was a no-go?
Or is this something else?

Reply:
Originally, I thought it wasn't possible, so I was going to add an early error in an 8.7 patch release. While implementing, I realized it was possible, so https://github.com/canonical/snapcraft/pull/5265 turned into addign functionality and was put into the 8.8 release.

You may be thinking of this unrelated early error case from last week: https://github.com/canonical/snapcraft/pull/5351